src/content/pages/test.mdx

```diff
@@ -0,0 +1,272 @@
+---
+title: test
```

Comment:
We could call it Kitchen Sink, like the Sphinx one :) 

https://pradyunsg.me/furo/kitchen-sink/

```suggestion
title: Kitchen Sink
```

Reply:
I love it! 🤩